app/src/main/res/layout/activity_data_overview.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".DataOverviewActivity">
+
+    <fragment
+        android:id="@+id/dataOverview_fragment"
+        class="com.github.HumanLearning2021.HumanLearningApp.DatasetOverViewFragment"
+        android:layout_width="300dp"
+        android:layout_height="500dp"
+        android:layout_marginStart="52dp"
+        android:layout_marginTop="112dp"
```

Comment:
it would really be better to try not to use fixed values as much as possible, so that the app adapts to different screen sizes. maybe this can help : https://www.youtube.com/watch?v=PqkWT92BT3U&t=1s